sdk/monitor/azure-monitor-ingestion/src/samples/java/com/azure/monitor/ingestion/UploadLogsAsyncClientSample.java

```diff
@@ -38,37 +37,26 @@ private void run() throws InterruptedException {
 
         CountDownLatch countdownLatch = new CountDownLatch(1);
         List<Object> dataList = getLogs();
-        try {
-
-            // More details on Mono<> can be found in the project reactor documentation at :
-            // https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html
-
-            Mono<UploadLogsResult> resultMono = client.upload("<data-collection-rule-id>",
+        // More details on Mono<> can be found in the project reactor documentation at :
+        // https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html
+        Mono<Void> resultMono = client.upload("<data-collection-rule-id>",
```

Comment:
nit (and not-blocking): would be nice to add an example for async error subscriber callback and error callback in upload options to demonstrate behavior in this case.

Reply:
added the sample